bitnami/grafana-loki/templates/query-scheduler/deployment.yaml

```diff
@@ -0,0 +1,165 @@
+{{- if .Values.queryScheduler.enabled }}
+apiVersion: {{ include "common.capabilities.deployment.apiVersion" . }}
+kind: Deployment
+metadata:
+  name: {{ template "grafana-loki.query-scheduler.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/part-of: grafana-loki
+    app.kubernetes.io/component: query-scheduler
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  minReadySeconds: 10
```

Comment:
Is there any requirement or reason to set `minReadySeconds`? If there is a reason to use it, could make sense to create a value that allow users change it

Reply:
I'm honestly not sure, it was generated by Tanka. I'll add it as a value.